### PR TITLE
runtimbp: Add shares fallback

### PR DESCRIPTION
**Pull request by fishy**

### CODE REVIEW

File: runtimebp/cpu.go [hunk 1]
```diff
@@ -55,6 +56,40 @@ func NumCPU() (n float64) {
...
+				"NumCPU: falling back to use runtime.NumCPU(): %v, %v\n",
+				n,
+				err,
+			)
+			n = float64(runtime.NumCPU())
+		}
+	}()
+
+	// Big enough buffer to read the number in the file wholly into memory.
+	buf := make([]byte, 1024)
```
Comment (konradreiche) [inline runtimebp/cpu.go]: Why not just do a `ioutil.ReadFile` and convert that?

Comment (fishy) [reply runtimebp/cpu.go]: the file could be huge, we don't want to blow up memory.

Comment (konradreiche) [reply runtimebp/cpu.go]: Doesn't that file only ever have a number in there? Well, doesn't matter, was just curious :)

Comment (fishy) [reply runtimebp/cpu.go]: It could be misconfigured somewhere.

**Pull request merged**